ethereum/p2p/src/test/java/tech/pegasys/pantheon/ethereum/p2p/discovery/internal/PeerDiscoveryControllerTest.java

```diff
@@ -973,83 +972,205 @@ public void shouldNotAddPeerInNeighborsPacketWithoutBonding() {
   }
 
   @Test
-  public void shouldNotBondWithNonPermittedPeer() {
```

Comment:
Just adding a note on these changes since the diff is confusing.  

The tests `shouldNotBondWithNonPermittedPeer` and `shouldNotRespondToPingFromNonWhitelistedDiscoveryPeer` were removed.  More comprehensive tests that include these cases were added to `PeerDiscoveryAgentTest`.   The new tests here are not related to the previous tests that were removed.